src/class.c

```diff
@@ -767,47 +771,132 @@ boot_defclass(mrb_state *mrb, struct RClass *super)
   return c;
 }
 
-MRB_API void
-mrb_include_module(mrb_state *mrb, struct RClass *c, struct RClass *m)
+static void
+boot_initmod(mrb_state *mrb, struct RClass *mod)
```

Comment:
I'm bad a naming, I tried to make it match the existing `boot_defclass` which is for bootstrapping the first 4 classes (BasicObject, Object, Class and Module). `bootstrap_initialize_module` is what its read as, but yeah, feel free to rename it, its not a class_alloc, since it takes a RClass module and initializes its fields